jbpm-xes/pom.xml

```diff
@@ -127,6 +127,24 @@
       <groupId>org.postgresql</groupId>
       <artifactId>postgresql</artifactId>
     </dependency>
+    
+    <dependency>
+      <groupId>org.jboss.spec.javax.xml.bind</groupId>
+      <artifactId>jboss-jaxb-api_2.3_spec</artifactId>
+      <scope>provided</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>com.sun.xml.bind</groupId>
+      <artifactId>jaxb-impl</artifactId>
+      <scope>test</scope>
+    </dependency>
+    
+    <dependency>
+      <groupId>com.sun.xml.bind</groupId>
+      <artifactId>jaxb-core</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
I think we can leave out the scope entirely (which means compile scope), since no other components depend on this one and if somebody wants to use the tool on JDK 11, they will need all these dependencies anyway.

Reply:
Good point, agreed. Also added a comment before for highlighting that this is needed because of jdk11